docs/anomaly-detection/QuickStart.md

```diff
@@ -57,9 +57,35 @@ docker run -it -v $YOUR_LICENSE_FILE_PATH:/license \
                --license-file=/license
 ```
 
+In case you found `PermissionError: [Errno 13] Permission denied:` in `vmanomaly` logs, set user/user group to 1000 in the run command above / in a docker-compose file:
+
+```sh
+export YOUR_LICENSE_FILE_PATH=path/to/license/file
+export YOUR_CONFIG_FILE_PATH=path/to/config/file
+docker run -it --user 1000:1000 \
+               -v $YOUR_LICENSE_FILE_PATH:/license \
+               -v $YOUR_CONFIG_FILE_PATH:/config.yml \
+               vmanomaly /config.yml \
+               --license-file=/license
+```
+
+```yaml
+# docker-compose file
+services:
+  # ...
+  vmanomaly:
```

Comment:
This should include the full compose needed to make VMAnomaly work so people can copy paste

```
  vmanomaly:
    image: victoriametrics/vmanomaly:latest
    volumes:
        $YOUR_LICENSE_FILE_PATH:/license
        $YOUR_CONFIG_FILE_PATH:/config.yml
    command:
      - "/config.yml"
      - "--license-file=/license"
```